### ISSUE TITLE
Automate unpack when downloads is used

### ISSUE THREAD

**Issue by lefred** [datacharmer/dbdeployer#81]
**Is your feature request related to a problem? Please describe.**
I think it will be more intuitive and convenient to unpack the binary just after downloading it. Most of the time pple downloading a version will be to use it right away. Maybe add an option to not do so. (and by default I would even delete the downloaded file after unpack).

**Describe the solution you'd like**
dbdeployer downloads get mysql-8.0.16-linux-x86_64-minimal.tar.xz

should download the binary, unpack and delete the downloaded file. Then the new version should be available directly in the output of `dbdeployer versions`

Grazie ;)

**Comment by datacharmer** [comment]
Thanks for your request.
It's a sensible one, but it has a few twists:

* What to do when you download a tarball that was already unpacked? We need an option to overwrite.
* What to do when the unpack needs a prefix (e.g. ndb, pxc, tidb) or/and an explicit version (e.g. tidb)? More options are needed.

While it is doable, the automation needs some refinement.

Regarding the deletion by default, it would be very much inconvenient if this happened while you are on the road and using expensive data plans. I prefer leaving the costly deletion to the user's responsibility.

The way I would do it is:
```
$ dbdeployer downloads get mysql-8.0.16-linux-x86_64-minimal.tar.xz \
    --unpack --delete

$ dbdeployer downloads get tidb-master-darwin-amd64.tar.gz \
     --unpack --prefix=tidb --unpack-version=3.0.0 --delete
```

You can then use  [dbdeployer built-in abbreviations](https://github.com/datacharmer/dbdeployer/blob/master/abbreviations/sample_abbreviations.txt) to convert the multiple options into a manageable single command.

**Comment by lefred** [comment]
Great thx

**Comment by datacharmer** [comment]
[Implemented in 1.33.0](https://github.com/datacharmer/dbdeployer#From-remote-tarball-to-ready-to-use-in-one-step)
